packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -1280,7 +1280,7 @@ export const RemoteServerFactory = {
                 new ApiGatewayV1Adapter({
                     // Preserve the original aws-serverless-express behavior
                     lowercaseRequestHeaders: true,
-                    throwOnChunkedTransfer: false
+                    throwOnChunkedTransferEncoding: false
```

Comment:
Thanks

I can confirm this is the right option name --> [link to code](https://github.com/H4ad/serverless-adapter/blob/d92d680fda10f365f92e78567f4201ce7b3145e8/src/adapters/aws/api-gateway-v1.adapter.ts#L45)